src/arch/aarch64/arch_musl.cpp

```diff
@@ -0,0 +1,27 @@
+#include <os>
+#include <errno.h>
+#include <musl/common.hpp>
+#include "cpu.hpp"
+
+extern "C"
+long sys_set_thread_area(struct user_desc *u_info)
+{
+  set_tpidr(u_info); // This probably still does not work
+  return 0;
+}
+
+extern "C"
+long syscall_SYS_set_thread_area(struct user_desc *u_desc) {
+    return strace(sys_set_thread_area, "set_thread_area", u_desc);
+}
+
+extern "C"
+long sys_arch_prctl(int code, uintptr_t ptr) {
+    os::panic("This should not happen!");
```

Comment:
Please explain why. Like because multithreading is not supported on this arch or something. As helpful as you can.